### PR TITLE
Clarify first use of "elide" in lifetime.md

### PR DESCRIPTION
**Pull request by corincerami**

### CODE REVIEW

**Comment by Gankro** [comment]
"elide" is standard community terminology for this family of behaviours, fwiw.

I think in this case, this is an improvement though.

@bors r+ rollup

**Comment by steveklabnik** [comment]
So: I _do_ agree that 'elide' is a weird word, but we also call it "lifetime elision" everywhere, and so, in some sense, using this word _aids_ discover-ability.

So I'd like to talk about it before we merge. I'm not sure what the right path of action here is, exactly.

**Comment by Gankro** [comment]
I would add a parenthetical mentioning the concept of "elision" here if that's really desirable.

I'd need to go over this section more to get the context, but I was under the impression that this was at a point where we're "not even ready" to discuss lifetime elision in earnest.

**Comment by chrisccerami** [comment]
That sounds like a reasonable idea. I think it makes sense to at least clarify the use of "elide", even if that just involves including something like "(see 'Lifetime Elision' below)" to the sentence.

**Comment by steveklabnik** [comment]
:+1: to both things. I think a forward reference here is just fine. Maybe something like 

> lets you elide (i.e. not write, see ["Lifetime Elision"](#) below) them in

?

**Comment by chrisccerami** [comment]
:+1: to that idea.

**Comment by steveklabnik** [comment]
Great! Amend this PR and we can :shipit:

**Comment by chrisccerami** [comment]
:shipit: :rainbow:

File: src/doc/trpl/lifetimes.md [hunk 1]
```diff
@@ -70,9 +70,11 @@ fn bar<'a>(x: &'a i32) {
 ```
 
 The `'a` reads ‘the lifetime a’. Technically, every reference has some lifetime
-associated with it, but the compiler lets you elide them in common cases.
+associated with it, but the compiler lets you elide (i.e. omit, see ["Lifetime Elision"][lifetime-elision] below) them in common cases.
```
Comment (steveklabnik) [inline src/doc/trpl/lifetimes.md]: can you wrap this to 80 chars please?

**Comment by chrisccerami** [comment]
:rainbow:

**Pull request merged**